bin/gunicorn_start

```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+
+ENVIRONMENT=$1
+NAME="openprescribing_{$ENVIRONMENT}"
+
+# INSTALL_ROOT and SITE_ROOT match definitions in settings
+INSTALL_ROOT=$(dirname $(dirname $0))
+SITE_ROOT="$INSTALL_ROOT/openprescribing"
+
+RUNDIR="$INSTALL_ROOT/run"
+LOGDIR="$INSTALL_ROOT/logs"
+
+# Create the run and logs directories if they don't exist
+mkdir -p $RUNDIR
+mkdir -p $LOGDIR
+
+SOCKFILE="$RUNDIR/gunicorn.sock"
+LOGFILE="$LOGDIR/gunicorn-error.log"
+ACCESS_LOGFILE="$LOGDIR/gunicorn-access.log"
+
```

Comment:
Maybe assert environment is one of the allowed values here?  e.g.

    [[ $ENV == "production" || $ENV == "staging" ]] || (echo '$ENV must be "production" or "staging"' && exit 1)

Reply:
Good idea, fixed.  (Although it turns out that `(` and `)` start a subshell, and the `exit 1` exits from the subshell, so I've rewritten slightly.)